lib/rtti.h

```diff
@@ -67,6 +67,9 @@ namespace RTTI {
 
 using TypeId = std::uint64_t;
 static constexpr TypeId InvalidTypeId = UINT64_C(0);
+static constexpr unsigned kDiscriminatorBits = 56;
+static constexpr uint64_t kInnerTypeIdMask = (UINT64_C(1) << kDiscriminatorBits) - 1;
+static constexpr uint64_t kHashDiscriminator = UINT64_C(0xFF);
```

Comment:
What does the `k` prefix mean? I don't think we use this naming convention anywhere in the code base.

Reply:
This is probably something we should encode in the `.clang-tidy` file. We have some style suggestions there already but they are fairly arbitrary.